recipes/raylib/all/conanfile.py

```diff
@@ -22,11 +23,43 @@ class RaylibConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "opengl_version": [None, "4.3", "3.3", "2.1", "1.1", "ES-2.0"],
+
+        "module_rshapes": [True, False],
+        "module_rtextures": [True, False],
+        "module_rtext": [True, False],
+        "module_rmodels": [True, False],
+        "module_raudio": [True, False],
+
+        "mouse_gestures": [True, False],
+        "default_font": [True, False],
+        "screen_capture": [True, False],
+        "gif_recording": [True, False],
+        "busy_wait_loop": [True, False],
+        "events_waiting": [True, False],
+        "compression_api": [True, False],
+        "events_automation": [True, False],
+        "custom_frame_control": [True, False]
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "opengl_version": None,
+
+        "module_rshapes": True,
```

Comment:
I would suggest adding only option that you really need for now, otherwise, it will increase the recipe maintenance. We can not validate each combination, which means there is a risk breaking things depending the combination.

I also would request a build log, if possible, using the options that you are adding, I mean, using the non-default option value, to make sure it's not broken. I asking it because we found recipes with custom options that are not working and the CI really does not check it (only shared, fPIC and header_only are checked), so any user opens an issue months later reporting about that option is broken and recipe/upstream is bugged.